assets/terms.txt

```diff
@@ -0,0 +1,111 @@
+
+SUBMISSIONS
+You acknowledge and agree that any questions, comments, suggestions, ideas, feedback, or other information regarding the App ("Submissions") provided by you to us are non-confidential and shall become our sole property. We shall own exclusive rights, including all intellectual property rights, and shall be entitled to the unrestricted use and dissemination of these Submissions for any lawful purpose, commercial or otherwise, without acknowledgment or compensation to you. You hereby waive all moral rights to any such Submissions, and you hereby warrant that any such Submissions are original with you or that you have the right to submit such Submissions. You agree there shall be no recourse against us for any alleged or actual infringement or misappropriation of any proprietary right in your Submissions.
+
+APP MANAGEMENT
+We reserve the right, but not the obligation, to: (1) monitor the App for violations of these Terms of Use; (2) take appropriate legal action against anyone who, in our sole discretion, violates the law or these Terms of Use, including without limitation, reporting such user to law enforcement authorities; (3) in our sole discretion and without limitation, refuse, restrict access to, limit the availability of, or disable (to the extent technologically feasible) any of your Contributions or any portion thereof; (4) in our sole discretion and without limitation, notice, or liability, to remove from the App or otherwise disable all files and content that are excessive in size or are in any way burdensome to our systems; and (5) otherwise manage the App in a manner designed to protect our rights and property and to facilitate the proper functioning of the App.
+
+PRIVACY POLICY
+We care about data privacy and security. Please review our Privacy Policy [CLICK HERE]. By using the App, you agree to be bound by our Privacy Policy, which is incorporated into these Terms of Use. Please be advised the App is hosted in the United States. If you access the App from the European Union, Asia, or any other region of the world with laws or other requirements governing personal data collection, use, or disclosure that differ from applicable laws in the United States, then through your continued use of the App or Services, you are transferring your data to the United States, and you expressly consent to have your data transferred to and processed in the United States. [Further, we do not knowingly accept, request, or solicit information from children or knowingly market to children. Therefore, in accordance with the U.S. Children’s Online Privacy Protection Act, if we receive actual knowledge that anyone under the age of 13 has provided personal information to us without the requiApp and verifiable parental consent, we will delete that information from the App as quickly as is reasonably practical.]
```

Comment:
It says [CLICK HERE] 

But it is not actually clickable.
Please edit.

It also says the app is hosted in the United States.
And the Children's privacy protection says below 13. Shouldn't we make it below 18 here too?
Please clarify

Reply:
Would fix that boss

Reply:
Life would be easier for me if you accepted your rightful place as the boss, and stopped shaming me in public with the title